src/app/component/comments/comments.module.ts

```diff
@@ -1,21 +1,20 @@
 import { NgModule } from '@angular/core';
-import {CommentsComponent} from './comments/comments.component';
-import {SharedModule} from '../shared/shared.module';
-import {CommonModule} from '@angular/common';
-import {CommentsRoutingModule} from './comments-routing.module';
-
+import { CommentsComponent } from './comments/comments.component';
+import { SharedModule } from '../shared/shared.module';
+import { CommonModule } from '@angular/common';
+import { AddCommentComponent } from '../comments/comments/add-comment/add-comment.component';
```

Comment:
is there any reason to use path with double name 'comments/comments'? Your module may well be located into the same folder with component.

Reply:
fixed